hooks/useRealm.tsx

```diff
@@ -25,9 +25,11 @@ export default function useRealm() {
     tokenRecords,
     councilTokenOwnerRecords,
   } = useWalletStore((s) => s.selectedRealm)
-  const realmInfo = useMemo(() => getRealmInfo(symbol as string, apiEndpoint), [
-    symbol,
-  ])
+
+  const realmInfo = useMemo(
+    () => getRealmInfo(symbol as string, connection.endpoint as EndpointTypes),
+    [symbol]
+  )
```

Comment:
This is the change to use the connection from useWalletStore for getting the realms and realmInfo, so that it all uses the same connection (set from query param)

Reply:
:+1: